library/agent/ServiceConfig.ts

```diff
@@ -126,6 +135,40 @@ export class ServiceConfig {
     return { blocked: false };
   }
 
+  private setOnlyAllowedIPAddresses(ipAddresses: IPList[]) {
+    this.onlyAllowedIPAddresses = [];
+
+    for (const source of ipAddresses) {
+      // Skip empty allowlists
+      if (source.ips.length === 0) {
+        continue;
+      }
+      this.onlyAllowedIPAddresses.push({
```

Comment:
Wasn't @hansott adding a key for this so we can report back statistics

Reply:
Yep, but neither core or node PR are merged yet, I think we could better update them?

Reply:
True